src/UnitySampleProject/Packages/manifest.json

```diff
@@ -45,8 +45,5 @@
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
     "com.unity.modules.xr": "1.0.0"
-  },
-  "testables": [
```

Comment:
We need to keep this for the package to be testable from the test runner and also CI